bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+/**
+ * Generate static design screenshot thumbnails for Gutenboarding
+ * Until we can generate tall screenshots on the fly with mshots:
+ * https://github.com/Automattic/mShots/issues/16
+ * https://github.com/Automattic/wp-calypso/issues/40564
+ *
+ * You'll need Puppeteer installed.
+ * Change `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`in `package.json` to `false` and run `npm ci`
+ */
+
+const designs = require( '../client/landing/gutenboarding/available-designs.json' );
+const puppeteer = require( 'puppeteer' );
+const wpUrl = require( '@wordpress/url' );
+
+const screenshotsPath = './static/page-templates/design-screenshots';
+
+const getDesignUrl = design => {
+	return wpUrl.addQueryArgs( design.src, {
+		font_headings: design.fonts[ 0 ],
+		font_base: design.fonts[ 1 ],
+	} );
+};
+
+async function run() {
+	console.log( 'Taking screenshots...' );
+	const browser = await puppeteer.launch( { headless: true } );
+
+	await Promise.all(
+		designs.featured.map( async design => {
+			const url = getDesignUrl( design );
+			const screenshotFile = `${ screenshotsPath }/${ design.slug }_${ design.template }_${ design.theme }.jpg`;
+
+			console.log( `Opening ${ url }` );
+			const page = await browser.newPage();
+			await page.setViewport( {
+				width: 980,
+				height: 2200,
+				deviceScaleFactor: 1,
+			} );
+			await page.goto( url, { waitUntil: 'networkidle0' } );
```

Comment:
Possible options for `waitUntil` would be 

> - `load` - consider navigation to be finished when the load event is fired.
> - `domcontentloaded` - consider navigation to be finished when the DOMContentLoaded event is fired.
> - `networkidle0` - consider navigation to be finished when there are no more than 0 network connections for at least 500 ms.
> - `networkidle2` - consider navigation to be finished when there are no more than 2 network connections for at least 500 ms.

via https://github.com/puppeteer/puppeteer/blob/3dff24ad35cd46754dbe0c77a9c7f7ade2454b88/docs/api.md#pagewaitfornavigationoptions